tardis/model/density.py

```diff
@@ -137,19 +137,19 @@ def calculate_density_after_time(densities, time_0, time_explosion):
     scale the density from an initial time of the model to the
     time of the explosion by ^-3
 
-    Parameters:
+    Parameters
     -----------
 
-    densities: ~astropy.units.Quantity
+    densities: astropy.units.Quantity
```

Comment:
Let's keep the space before the `:` consistent. There are other instances of this too.